src/cmd/sanity_test/export/onedrive.go

```diff
@@ -0,0 +1,86 @@
+package export
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	msgraphsdk "github.com/microsoftgraph/msgraph-sdk-go"
+
+	"github.com/alcionai/corso/src/cmd/sanity_test/restore"
+	"github.com/alcionai/corso/src/cmd/sanity_test/utils"
+	"github.com/alcionai/corso/src/internal/common/ptr"
+)
+
+func CheckOneDriveExport(
+	ctx context.Context,
+	client *msgraphsdk.GraphServiceClient,
+	userID, folderName, dataFolder string,
+) {
+	drive, err := client.
+		Users().
+		ByUserId(userID).
+		Drive().
+		Get(ctx, nil)
+	if err != nil {
+		utils.Fatal(ctx, "getting the drive:", err)
+	}
+
+	// map itemID -> item size
+	fileSizes := make(map[string]int64)       // exportFileSizes = make(map[string]int64)
+	exportFileSizes := make(map[string]int64) // exportFileSizes = make(map[string]int64)
+	startTime := time.Now()
```

Comment:
Can the comments get removed?

```suggestion
        var (
	  fileSizes = make(map[string]int64)       // exportFileSizes = make(map[string]int64)
	  exportFileSizes = make(map[string]int64) // exportFileSizes = make(map[string]int64)
	  startTime = time.Now()
	)
```